src/app/dataset-view/additional-components/overview-component/components/edit-watermark-modal/edit-watermark-modal.component.scss

```diff
@@ -1,9 +1,9 @@
 .custom-picker {
-    z-index: 9999 !important;
+    z-index: 9999;
 }
 
 .w-65 {
-    width: 67% !important;
+    width: 67%;
```

Comment:
It seems really strange :D

Class w-65 with the value 67
Could you please rename it have a garmony?

Reply:
Fixed